.github/workflows/e2e.yml

```diff
@@ -26,4 +26,4 @@ jobs:
       run: sudo sysctl -w net.netfilter.nf_conntrack_max=131072
 
     - name: Run PR-Blocking e2e tests
-      run: GINKGO_FOCUS="\[PR-Blocking\]" make test-e2e
+      run: yes | GINKGO_FOCUS="\[PR-Blocking\]" make test-e2e
```

Comment:
Why is this listed as a change? We already have this on `main`. Maybe something wrong with rebase?
https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/2fc856d5f122fad4c8ddc95d023a6f0e14c16059/.github/workflows/e2e.yml#L28-L29